concent_api/core/models.py

```diff
@@ -306,6 +306,9 @@ def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
         self._current_state_name = None
 
+    def __repr__(self):
+        return f"Subtask: task_id={self.task_id}, subtaks_id={self.subtask_id}, state={self.state_enum}"
```

Comment:
This should be `__str__()`, not `__repr__()`. [repr()](https://docs.python.org/3/library/functions.html#repr) is supposed to return a reprsentation that can be evaluated to get the original object back. `str()` can be more lax.

Reply:
Not really. 
1) `__repr__` :
> If at all possible, this should look like a valid Python expression that could be used to recreate an object with the same value (given an appropriate environment).
In this case it is not. 
2) The most important trait of `__repr__` is that it should be unambiguous. Printing information 
about `task_id` and `subtask_id`  is enough to identify a Subtask object.
3) `__str__` of collection (like `QuerySet`) uses `__repr__` of objects it contains (and not their `__str__`).
4) If an object doesn't have `__str__` defined, it will use its `__repr__`, but not vice versa.

**TL;DR**
The use of `__repr__` was intentional.

https://stackoverflow.com/a/2626364

Reply:
Fair enough. The part about it only having to be unambiguous seems convincing.

But maybe let's tweak the representation a bit so it looks like `Subtask(task_id=1, subtask_id=2, state=3)`? Otherwise it may be hard to read if you print a list of subtasks.